packages/neon-js/src/experimental/helpers.ts

```diff
@@ -208,3 +232,55 @@ export async function addFees(
     );
   }
 }
+
+/**
+ * Deploy a smart contract
+ * @param NEF - A smart contract in Neo executable file format. Commonly created by a NEO compiler and stored as .NEF on disk
+ * @param manifest - the manifest conresponding to the smart contract
+ * @param config -
+ */
+export async function deployContract(
+  NEF: Buffer,
+  manifest: sc.ContractManifest,
+  config: CommonConfig
+): Promise<string> {
+  const builder = new sc.ScriptBuilder();
+  builder.emitAppCall(CONST.NATIVE_CONTRACT_HASH.ManagementContract, "deploy", [
+    u.HexString.fromHex(u.reverseHex(NEF.toString("hex"))),
+    JSON.stringify(manifest.toJson()),
```

Comment:
Is this emitting it as a Map ContractParam? Im not sure if I implemented the support correctly.

Reply:
I'm guessing this emits an Array, honestly I did not look into what it creates. I just verified it is valid on the C# side.

Reply:
yeah we have to test this out definitely i have not implemented support so I assume it breaks.

Reply:
I'm not sure what extra you want to test regarding this emitting? I tested that the `deployContract` function works as intended. I know for sure it deserializes correctly on the C# side, because initially I
1) did not have the `u.reverseHex()` on the NEF parameter which caused the engine to fail deserializing the NEF. 
2) the manifest was missing `safe` and `name` attributes,  causing the engine to fail deserializing the manifest.
So to me that tells me that it is emitting what it needs to emit?